src/utils/hooks/Base64.js

```diff
@@ -0,0 +1,3 @@
+export const decode = (base64) => new Buffer(base64, "base64").toString('ascii')
+
+export const encode = (string) => new Buffer(string).toString('base64')
```

Comment:
może `base64decode` i `base64encode`, bo widać że to base64 w imporcie, ale w kodzie już kiepsko